compass/ocean/tests/drying_slope/initial_state.py

```diff
@@ -28,6 +28,8 @@ def __init__(self, test_case):
 
         self.add_namelist_file('compass.ocean.tests.drying_slope',
                                'namelist.init', mode='init')
+        self.add_namelist_file('compass.ocean.tests.drying_slope',
+                               f'namelist.{coord_type}.init', mode='init')
```

Comment:
Thanks for your thoughts on this. I agree. The issue you discuss here is addressed by https://github.com/MPAS-Dev/compass/pull/370/commits/21ff37a82c537a7a71abe5aa12f8a598822e9a2e. This does ignore the `vert_levels` option when the coordinate type is `single_layer`. In the process, I realized that specifying the coordinate type in the `drying_slope.cfg` file also made no sense since this should be inherited from the choice of coordinate system. I made those changes here: https://github.com/MPAS-Dev/compass/pull/370/commits/a0c82b96672f00413348fbc4c1b79b74c340077c